programs/amm/Cargo.toml

```diff
@@ -21,4 +22,5 @@ overflow-checks = true
 [dependencies]
 anchor-lang = "0.24.2"
 anchor-spl = "0.24.2"
-mercurial-vault = { git="https://github.com/mercurial-finance/vault-sdk", features= ["cpi"], rev = "6daf57a877c36694c8f5b8267ca1e096a1e619d2" }
+mercurial-vault = "0.4.1"
```

Comment:
We need CPI feature here
`mercurial-vault = { version = "0.4.1", features = ["cpi"] }`